pkg/management/component.go

```diff
@@ -710,14 +710,15 @@ func (c *component) ResetSmsCounter(ctx context.Context, realmName, userID strin
 		return err
 	}
 
-	//reset the counter, if the smsSent attribute exists
+	//reset the counters, if the smsSent or smsAttempts attributes exist
 	resetCounter := 0
 	if userKc.Attributes != nil {
 		var m = *userKc.Attributes
-		if m["smsSent"] != nil {
+		if m["smsSent"] != nil || m["smsAttempts"] != nil {
 			// ensure there is no unencrypted PII
 			keycloakb.ConvertLegacyAttribute(&userKc)
 			(*userKc.Attributes)["smsSent"][0] = strconv.Itoa(resetCounter)
+			(*userKc.Attributes)["smsAttempts"][0] = strconv.Itoa(resetCounter)
```

Comment:
You should be able to write something like:
userKc.SetAttributeInt(constants.AttrbSmsSent, 0)
userKc.SetAttributeInt(constants.AttrbSmsAttempts, 0)